ci/install_node_npm.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
-curl -sSL "https://nodejs.org/dist/v11.5.0/node-v11.5.0-linux-x64.tar.xz" | sudo tar --strip-components=2 -xJ -C /usr/local/bin/ node-v11.5.0-linux-x64/bin/node
+curl -sSL "https://nodejs.org/dist/v13.8.0/node-v13.8.0-linux-x64.tar.xz" | sudo tar --strip-components=2 -xJ -C /usr/local/bin/ node-v13.8.0-linux-x64/bin/node
```

Comment:
We generally use node:lts (long term support) elsewhere in ci. For consistency, can we use that here? Is there an lts link? If not, 12.16.2 is the current LTS version.

Reply:
Good catch, I'll peg it to the latest LTS. I haven't been able to find a URL for the default LTS

Reply:
That sucks. I wish they had a standard LTS link, like they do for docker images.